### ISSUE TITLE
Toolbar buttons don't provide a focus outline [SC 2.4.7]

### ISSUE THREAD

**Issue by JeffersonBledsoe** [plone/volto#6311]
https://www.w3.org/TR/WCAG22/#focus-visible

Currently we don't have any focus indicator for the buttons in the toolbar. This is confusing when using a keyboard to navigate the site and is a failure of the WCAG 2.X success criterion noted in the title
